### PR TITLE
added PM-IO-5-SM

### PR DESCRIPTION
**Pull request by J2thatsme**
No tests @ the moment with this documentation.
But feedback is welcomed ^^

### CODE REVIEW

File: types/tinovi/PM-IO-5-SM/default.schema.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Default",
+  "type": "object",
+  "properties": {
+    "valve": {
+      "title": "Valve",
+      "type": "boolean",
+      "description": "Valve on = true, Valve off = false"
+    },
+    "leak": {
```
Comment (mariusgiger) [inline types/tinovi/PM-IO-5-SM/default.schema.json:12]: better use leakDetected

File: types/tinovi/PM-IO-5-SM/default.schema.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      "maximum": 3276.5
+    },
+    "waterContent": {
+      "title": "Water content",
+      "unit": "%",
+      "type": "number",
+      "description": "Water content in the soil",
+      "minimum": 0,
+      "maximum": 100
+    }
```
Comment (mariusgiger) [inline types/tinovi/PM-IO-5-SM/default.schema.json:38]: there are quite a few missing data keys

File: types/tinovi/PM-IO-5-SM/default.schema.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    "temperature": {
+      "title": "Temperature",
+      "unit": "°C",
+      "type": "number",
+      "description": "The temperature in °C.",
+      "minimum": -3276.5,
+      "maximum": 3276.5
+    },
+    "waterContent": {
+      "title": "Water content",
```
Comment (mariusgiger) [inline types/tinovi/PM-IO-5-SM/default.schema.json:32]: Volumetric water content

**Review by mariusgiger — COMMENTED** [review]
looks good, some minor comments, please add a test once you have got an example payload, also add a changelog entry

File: types/tinovi/PM-IO-5-SM/default.schema.json (hunk 2 above)
Comment (J2thatsme) [reply types/tinovi/PM-IO-5-SM/default.schema.json:38]: These keys are the only ones for this sensor. I could add the rest but we said we would make different devicetypes for different sensors even if they may have the same decoder.

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**